packages/cli/lib/pipeline.js

```diff
@@ -20,14 +25,19 @@ const discoverOperations = async () => {
   return ops
 }
 
-export const desugar = async (dataset, { logger, knownOperations } = {}) => {
-  knownOperations = knownOperations ?? await discoverOperations()
+export const desugar = async (dataset, { logger, knownOperations, pipelinePath } = {}) => {
+  knownOperations = knownOperations ?? await discoverOperations(pipelinePath)
+  const dir = await packageDirectory({
+    cwd: pipelinePath,
+  })
+  const require = module.createRequire(dir + '/')
```

Comment:
Somehow, the base dir to require must be the path, ending in `/`. This is only important in global context